Sources/Plasma/FeatureLib/pfLocalizationMgr/pfLocalizationDataMgr.cpp

```diff
@@ -547,12 +538,13 @@ void LocalizationDatabase::IVerifyElement(const ST::string &ageName, const ST::s
             curTranslation++;
     }
 
-    for (const auto& language : languageNames)
-    {
-        if (theElement.find(language) == theElement.end())
-        {
+    for (auto lang : plLocalization::GetAllLanguages()) {
+        if (
+            plLocalization::IsLanguageUsable(lang)
+            && theElement.find(plLocalization::GetLanguageName(lang)) == theElement.end()
+        ) {
```

Comment:
This is difficult to parse, IMO.

Reply:
I can pull `plLocalization::GetLanguage(lang)` out into a variable I suppose?